nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,70 @@
+const express = require("express");
+const app = express();
+module.exports = app;
+// import data here
+const meals = require("./data/meals");
+const reservations = require("./data/reservations");
+const reviews = require("./data/reviews");
+const mealsNum = meals.map((meal) => meal.id);
+const reviewMealId = reviews.map((review) => review.mealId);
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("asd");
+});
+// meals with review
+app.get("/meals", async (request, response) => {
+  const mealWithReview = meals.map(meal => {
+    const reviewMatchMeal = reviews.filter(review => review.mealId === meal.id)
```

Comment:
Nitpick(variable naming):
I would have named these `mealsWithReviews` since it's multiple meals and each meal might have multiple reviews.
`reviewMatchMeal` I would call `reviewsForMeal`, again because it's multiples.
This applies to more variables in this file.